src/js/bootstrap-slider.js

```diff
@@ -1659,6 +1659,9 @@ const windowIsDefined = (typeof window === "object");
 				this._setDataVal(val);
 				this._trigger('slideStop', val);
 
+				// No longer need 'dragged' after mouse up
+				delete this._state.dragged;
```

Comment:
generally im not a big fan of removing properties with the `delete` keyword (it has dangling memory reference issues in multiple browsers).

Can you refactor this to just set dragged to `null` or something like that instead?

Reply:
I was just copying the behaviour of the keydown event, that will set and delete `this._state.keyCtrl`. I will set it to null, it doesn't seem to break any of the logic.

https://github.com/seiyria/bootstrap-slider/blob/2272d5a0249b6ccb05efc2d5eced52720d8272aa/src/js/bootstrap-slider.js#L1565